x-pack/plugins/observability_solution/apm/public/components/app/service_overview/service_overview_instances_table/instance_actions_menu/menu_sections.ts

```diff
@@ -25,14 +25,14 @@ type InstaceDetails =
 
 function getInfraMetricsQuery(timestamp?: string) {
   if (!timestamp) {
-    return { from: 0, to: 0 };
+    return undefined;
```

Comment:
Q: why did we change this?

Reply:
It is to match the new format for the assetDetails `dateRange`, since the `0` values might be problematic.